lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.html

```diff
@@ -1,49 +0,0 @@
-<form [formGroup]="form" novalidate (ngSubmit)="apply(form.value, form.valid)">
-    <mat-form-field>
-        <mat-label>{{ 'SEARCH.FILTER.RANGE.SELECT-FROM-DATE' | translate }}</mat-label>
-        <input matInput
-                (input)="forcePlaceholder($event)"
-                [formControl]="from"
-                [errorStateMatcher]="matcher"
-                placeholder="{{ 'SEARCH.FILTER.RANGE.FROM-DATE' | translate }}"
-                [attr.aria-label]="'SEARCH.FILTER.RANGE.FROM-DATE' | translate"
-                [matDatepicker]="fromDatepicker"
-                [max]="fromMaxDate"
-                (dateChange)="onChangedHandler($event, from)"
-                data-automation-id="date-range-from-input">
```

Comment:
In ACA there is variable which seems referencing to this removed element, please remove that variable: 
<img width="1312" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/3b7c72df-6b32-48a5-b24c-ff0fcb09ea81">

Reply:
There are still some minor default configuration changes required in ACA/ADW for this ticket. I'll remove the reference when raising the PR there. For now, I've added a [comment](https://alfresco.atlassian.net/browse/ACS-6107?focusedCommentId=1214499) on the JIRA  linking  to your comment here